python/cugraph/cugraph/dask/centrality/katz_centrality.py

```diff
@@ -109,7 +110,8 @@ def katz_centrality(input_graph,
         acceptable.
 
     nstart : dask_cudf.Dataframe, optional (default=None)
-        GPU Dataframe containing the initial guess for katz centrality
+        GPU Dataframe containing the initial guess for katz centrality.
```

Comment:
`Distributed` GPU dataframe

Reply:
It looks like `nstart` is not reported. Can you add a FIXME?

Reply:
nstart is now supported in the refactored MG Katz binding, so this was resolved when I moved it up to stable and removed old MG Katz